tests/calendars/test_gregorian.py

```
@@ -13,10 +13,53 @@
+)
 
 
 class TestGregorianCalendar(TestCase):
+    def test_get_nth_week_of_month(self):
+        # Week 1...4.
+        self.assertEqual(_get_nth_week_of_month(1, 1, 2024), date(2024, 1, 1))
+        self.assertEqual(_get_nth_week_of_month(2, 1, 2024), date(2024, 1, 8))
+        self.assertEqual(_get_nth_week_of_month(3, 1, 2024), date(2024, 1, 15))
+        self.assertEqual(_get_nth_week_of_month(4, 1, 2024), date(2024, 1, 22))
+        # No Week 5 in Jan 2024.
+        with self.assertRaises(ValueError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)